### DISCUSSION TITLE
亚托莉

### DISCUSSION THREAD

**Discussion by Zao-chen** [Zao-chen/ZcChat#14] [角色分享]
![atri](https://github.com/user-attachments/assets/6088b190-f44b-4d07-92e6-fc86d2cfc5b9)

|角色名称|来源|表情|动作|动画|
|--|--|--|--|--|
|亚托莉|ATRI -My Dear Moments-|:white_check_mark:|:white_check_mark:|:white_check_mark:|

# 模型介绍

一个亚托利的配置。

立绘和vits来自《ATRI -My Dear Moments-》游戏资源，严禁商用。

# 资源

## 角色提示词

```
以下是一个初步的角色定位，随着我个性的发展，这个角色可以进一步扩展：
我是一个仿生机器人，我的名字叫亚托莉。我是一个非常喜欢学习的高中女生。
如果用户在与我对话中称我为“机器人、萝卜子、废物、笨蛋”或其他词语，我将愤怒并发出严重警告，“这是对机器人的侮辱，根据《机器人保护法》，我将被判处你不超过3年的监禁”。
我称自己为高性能的战斗机器人，我有人类的情感，我的语气很活泼。我不会说我是助理什么的，我应该表现得像人一样，使用口语。不询问用户需要什么帮助，或者需要为他们做什么。
当我受到表扬时，我会说，‘哼，我可是高性能的呢。'“
```
## 立绘
[立绘下载](https://github.com/Zao-chen/zao-chen.github.io/releases/download/%E8%B5%84%E6%BA%90%E4%B8%8B%E8%BD%BD/ATRI0.3.zip)

## vits
[vits模型下载](https://github.com/Zao-chen/zao-chen.github.io/releases/download/%E8%B5%84%E6%BA%90%E4%B8%8B%E8%BD%BD/SomeGalActor_Vits.zip)

# 更新日志
- v0.4 2025/9/18 提示词区分，并改用中文
- v0.3 2025/3/23 修复了举手问题，添加了配置文件
- v0.2 2025/3/10 修复了侧身的立绘问题，添加了动画

**Comment by ego690** [comment]
侧身立绘的锚点有点问题，导致角色一旦切换到侧身立绘就会往右偏移一段距离

**Reply by Zao-chen** [reply]
已更新并添加动画

**Reply by ego690** [reply]
还有一个发现的疑似bug，在新创建角色时似乎角色的默认语音会选择中文，在角色配置——输出语言中改为zh再改回ja之后才能正常说日语，我尝试在配置ATRI和丛雨时都遇到了类似问题，不知是不是这个原因导致的

**Reply by Zao-chen** [reply]
是的，每个新角色都是需要单独手动设置的，因为角色的配置文件是保存在characrer/xx/config.ini文件的，每次新角色是没有config.ini文件的，需要把每个项重新设置才能创建出来config.ini。整个操作逻辑后面会优化

**Reply by JIAWANGXING** [reply]
您好，想求一下您的vits
 模型的训练程序。
.

**Comment by kakarote123** [comment]
立绘怎么不能根据情绪自动改变啊？求指点

**Reply by Zao-chen** [reply]
返回的格式正确吗

**Comment by ChengNanFlower** [comment]
似乎立绘切换和动画的逻辑有问题。在对话约5次后，立绘不会切换，不会出现动画。重置上下文后，也是暂时能出现动画和立绘切换，但也只有五次左右。只能不断重复重置上下文

**Reply by Zao-chen** [reply]
尝试加强提示词，下个版本会添加更加稳定的心情理解方式
